### DISCUSSION TITLE
Running dagster dev on specific host

### DISCUSSION THREAD

**Discussion by jcauserjt** [dagster-io/dagster#12219] [Q&A]
Hi --

Is it possible to specify a host in a `dagster dev` command? I saw an option for a port but not a host

I am looking for something equivalent to the `--host` option from the `dagit` command. Ultimately dagster dev launches a dagit (and a daemon) so hopefully there is a way for me to configure this.

Thanks

**Comment by jcauserjt** [comment]
Just wanted to follow-up on this. Thank you!

**Reply by sryza** [reply]
Hey @jcauserjt - there's a `--dagit-host` option.  CC @gibsondan .

**Reply by jcauserjt** [reply]
right, ive used the dagit command. My question is if there is a way to start both a daemon and dagit from a single command (like `dagster dev`) in a specific host. I basically want the same functionality offered in `datit` in `dagster dev` (or at least the part of specifying a host)

**Reply by gibsondan** [reply]
The --dagit-host/-h option is being added to the `dagster dev` command in the release tomorrow, at which point it should work just like dagit.

**Reply by sryza** [reply]
Oops - my suggestion was based on looking at the current code, didn't realize it hadn't been shipped yet
